ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataConnections.resource

```diff
@@ -103,7 +106,7 @@ Fill Data Connection Form
         Log    msg=you are not connecting any workbenchs to ${dc_name} DataConnection
     ELSE
         Run Keyword And Continue On Failure
-        ...    Element Should Be Enabled    xpath=//div[contains(@class,"modal")]//button[@aria-label="Options menu"]
+        ...    Element Should Be Enabled    xpath=//span[contains(@class, "pf-v5-c-text-input-group__text")]
```

Comment:
could you avoid using the patternfly version on the xpath? it's subject to change

Reply:
Thanks for the help, used your suggestions and they worked.